commands/handle_start_command.js

```diff
@@ -0,0 +1,23 @@
+const interaction = require('./../interaction');
+
+const escape = text => text.replace(/[_[\]()~`>#+\-=|{}.!\\]/g, '\\\$&');
+
+/** Handle connect command
+
+  Syntax of command:
+
+  /connect
+
+  {
+    from: <Message From User Id Number>
+    [id]: <Connected User Id Number>
+    reply: <Reply Function>
+  }
+*/
+module.exports = ({id, reply}) => {
+  if (!!id) {
+    return reply(escape(interaction.bot_is_connected));
+  }
+
+  return reply(escape(interaction.start_message));
```

Comment:
doesn't escaping the entire string also escape the markdown that is in the interaction message?

start message = `"🤖 *Bot started, run /connect to authorize.*"`

So escape would be escaping * because those characters are markdown?

I feel like they want you to do this: `🤖 *${escape('Bot started, run /connect to authorize.')}*` so that the . is escaped?

Reply:
ah but if there is * in the content you do want it to get escaped, I think that's the idea of escaping - just the way I was doing it before was wrong because I mixed markup and content together

Reply:
How do you want to handle this? We could also manually add escaping in the interactions I guess.